lldb/source/Core/Debugger.cpp

```diff
@@ -1210,14 +1212,18 @@ void Debugger::RestoreInputTerminalState() {
   {
     std::lock_guard<std::mutex> guard(m_statusline_mutex);
     if (m_statusline)
-      m_statusline->Enable();
+      m_statusline->Enable(GetSelectedExecutionContext());
   }
 }
 
-void Debugger::RedrawStatusline(bool update) {
+void Debugger::RedrawStatusline(
+    std::optional<ExecutionContextRef> exe_ctx_ref) {
   std::lock_guard<std::mutex> guard(m_statusline_mutex);
-  if (m_statusline)
-    m_statusline->Redraw(update);
+
```

Comment:
Are you missing a commit or something?  You have worked to pass yourself an exe_ctx_ref, but then you don't use it.

Reply:
You're correct, that's a remnant from when I was in the process of piping through the new context. This should use the one passed in.